kousa/mix.exs

```diff
@@ -21,16 +22,18 @@ defmodule Kousa.MixProject do
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:amqp, "~> 1.0"},
+      # TODO: consider switching to Registry
```

Comment:
doesn't this use Registry underneath the hood? What would the advantage of using Registry directly?

Reply:
it's just a bias, to not use libraries that are such a thin shim over stdlib features.

1 - blocks learning about the stdlib
2 - adds a barrier for contributors to immediately grok
3 - upgrade pathways to more advanced things (you will soon ;-) have to pick swarm or horde - i recommend horde) and you will have to decouple first and then upgrade.

Reply:
anyways it's not super important right now, plus you want tests on everything before you refactor =P